fission/src/test/mirabuf/MirabufParser.test.ts

```diff
@@ -5,6 +5,30 @@ import { mirabuf } from "../../proto/mirabuf"
 
 describe("Mirabuf Parser Tests", () => {
     test("Generate Rigid Nodes (Dozer_v9.mira)", async () => {
+        const expectedTransforms = [
```

Comment:
IDK if this is something we want to do now but this seems like the perfect application for [vitest snapshots](https://vitest.dev/guide/snapshot), which is their solution for making sure the output is the same after changes (rather than manually pasting the output of the functions)